src/components/Navbar/Toolbar.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars, faTimes } from '@fortawesome/free-solid-svg-icons';
+import { Link, useLocation } from 'react-router-dom';
+import PropTypes from 'prop-types';
+const Toolbar = ({ navbarOpen, handleToggleMenuClick }) => {
+  const location = useLocation();
+  const logoColor =
+    location.pathname === '/home' || location.pathname === '/about'
+      ? 'text-secondary'
+      : 'text-darkgray';
+  const navsColor =
+    location.pathname === '/home' || location.pathname === '/about'
+      ? 'text-secondary'
+      : 'text-primary';
+  const hoverEffect =
+    location.pathname === '/home' || location.pathname === '/about'
+      ? 'bg-primary'
+      : 'bg-darkgray';
+  return (
+    <React.Fragment>
```

Comment:
lets use this instead
```suggestion
    <>
```